pom.xml

```diff
@@ -536,6 +532,11 @@
         <artifactId>jsqlparser</artifactId>
         <version>${version.lib.jsqlparser}</version>
       </dependency>
+      <dependency>
+        <groupId>com.googlecode.json-simple</groupId>
```

Comment:
Out of curiosity, why do we need json-simple? Can't we do the same with jackson? I havn't been able to track down its use.

Reply:
This might be a leftover from the untyped json schema for tileset that has been reverted. Anyways i removed it.